### PR TITLE
docs: expand about section and add to main readme

### PR DESCRIPTION
**Pull request by Midnighter**
[skip CI]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,20 @@
 
 _TAXonomic Profile Aggregation and STAndardisation_
 
+## About
+
+The main purpose of taxpasta is to _standardise_ taxonomic profiles created by a
+range of bioinformatics tools. We call those tools taxonomic profilers. They
+each come with their own particular tabular output format. Relative abundances
+in read counts, fractions, or percentages. Any number of columns with additional
+information. We therefore decided to take [the lessons
```
Comment (jfy133) [inline README.md:17]:
```suggestion
each come with their own particular tabular output format. Across the profilers, relative abundances can be reported
in read counts, fractions, or percentages, as well as any number of additional columns with extra
information. We therefore decided to take [the lessons
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,20 @@
...
+learnt](https://xkcd.com/927/) to heart and provide our own solution to deal
```
Comment (jfy133) [inline README.md:18]: hehe

File: README.md [hunk 3]
```diff
@@ -8,6 +8,20 @@
...
+with this pasticcio. With taxpasta you can ingest all of those formats and, at
```
Comment (jfy133) [inline README.md:19]: What is pasticcio?

File: README.md [hunk 4]
```diff
@@ -8,6 +8,20 @@
...
+a minimum, output taxonomy identifiers and their integer counts.
+
+Taxpasta can not only standardise profiles but also _merge_ them
+across samples for the same profiler. In future, we also intend to offer methods
+for forming a _consensus_ for the same sample analyzed by different profilers.
 ## Install
```
Comment (jfy133) [inline README.md:25]:
```suggestion

## Install
```

File: docs/index.md [hunk 5]
```diff
@@ -4,11 +4,18 @@ _TAXonomic Profile Aggregation and STAndardisation_
 
 ## About
 
-The main purpose of taxpasta is to _standardise_ taxonomic profiles created by a range of
-bioinformatics tools. We call those tools taxonomic profilers. Taxpasta can not only
-standardise profiles but also _merge_ them across samples for the same profiler. In future,
-we also intend to offer methods for forming a _consensus_ for the same sample analyzed by
-different profilers.
+The main purpose of taxpasta is to _standardise_ taxonomic profiles created by a
```
Comment (jfy133) [inline docs/index.md:7]: Copy my comments from above

File: README.md (hunk 3 above)
Comment (Midnighter) [reply README.md:19]: Italian informally means 'mess'. I thought it fits well with taxpasta.

Comment (jfy133) [reply README.md:19]: haha love it!

Comment (sofstam) [reply README.md:19]: Pasticcio or greek lasagna 😆

**Review by jfy133 — APPROVED** [review]

**Pull request merged**